src/modules/bifrost/geometry/BifrostGeometry.cpp

```diff
@@ -63,10 +64,13 @@ int BifrostGeometry::yOffset(int Group) {
 
 std::pair<int, double> BifrostGeometry::calcUnitAndPos(int Group, int AmpA,
                                                        int AmpB) {
-
-  if (AmpA + AmpB == 0) {
-    XTRACE(DATA, DEB, "Sum of amplitudes is 0");
-    Stats.AmplitudeZero++;
+  if (int pulse_height = AmpA + AmpB; 0 == pulse_height || pulse_height > MaxAmpl){
```

Comment:
I would prefer to split this line in two different checks as it makes it more readable (IMO)

also I don't like the combination of declaration, assignment and logical check in the sme line :-)

Reply:
This is a reasonable request; will do.